addons/l10n_cl/models/account_move.py

```diff
@@ -27,11 +26,16 @@ def _get_l10n_latam_documents_domain(self):
         domain = super()._get_l10n_latam_documents_domain()
         if (self.journal_id.l10n_latam_use_documents and
                 self.journal_id.company_id.country_id == self.env.ref('base.cl')):
-            if self.type in ['in_invoice', 'in_refund']:
-                if self.partner_id.l10n_cl_sii_taxpayer_type == '2':
-                    domain += [('code', '=', '71')]
-                return domain
             document_type_ids = self.journal_id.l10n_cl_sequence_ids.mapped('l10n_latam_document_type_id').ids
+            document_type_codes = [x.l10n_latam_document_type_id.code for x in self.journal_id.l10n_cl_sequence_ids]
+            if self.partner_id_vat == '60805000-0' and '914' in document_type_codes:
+                return [('code', '=', '914')]
+            if self.type in ['in_invoice', 'in_refund']:
+                if self.partner_id.l10n_cl_sii_taxpayer_type == '2' and '71' in document_type_codes:
+                    return domain + [('code', '=', '71')]
+                if not self.partner_id.l10n_cl_sii_taxpayer_type or self.partner_id.country_id != self.env.ref(
+                        'base.cl') or self.partner_id.l10n_cl_sii_taxpayer_type == '4':
+                    return [('code', 'in', [])]
```

Comment:
So, it will give no results?  In the case of a foreigner, we don't need a document type.  What number is given to an invoice for a foreigner?

Reply:
for in_invoice or in_refund, (line 33) you don't need results for the document (there is no document available for this case). The given number is none in l10n_cl